RetailCoder.VBE/Inspections/VariableRequiresSetAssignmentEvaluator.cs

```diff
@@ -0,0 +1,160 @@
+﻿using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Inspections
+{
+    class VariableRequiresSetAssignmentEvaluator
```

Comment:
This class wants to be a static.  The only state that it holds is a `RubberduckParserState`, and that isn't even a true dependency - it's only being used to call `AllUserDeclarations`.  Taking that a step further, couldn't this just be rolled into the DeclarationFinder?